module/os/linux/zfs/abd_os.c

```diff
@@ -802,264 +701,45 @@ abd_alloc_for_io(size_t size, boolean_t is_metadata)
 	return (abd_alloc(size, is_metadata));
```

Comment:
If you feel like it, we should rewrite the above comment to just talk about Linux, since clearly the code in os/linux is  specific to linux, and if Illumos support is added to the common repo, it would get its own abd_os.c.

Reply:
Sounds good. I will update this comment.